integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -439,6 +426,60 @@ def export(self, **kwargs):  # pylint: disable=R0915
             enterprise_customer_identifier=self.enterprise_customer.name
         )
 
+    def _filter_out_pre_transmitted_enrollments(
+            self,
+            enrollment_queryset,
+            channel_name,
+            grade,
+            transmission_audit
+    ):
+        """
+        Given an enrollment_queryset, returns only enrollments that are not already transmitted
+        """
+        included_enrollments = set()
+        for enterprise_enrollment in enrollment_queryset:
+            if transmission_audit and \
+                    is_already_transmitted(transmission_audit, enterprise_enrollment.id, grade):
+                # We've already sent a completion status for this enrollment
+                generate_formatted_log(
```

Comment:
for now we can keep this log per item, I want to revisit this as part of the logging cleanup